tgui/packages/tgui/interfaces/IntegratedCircuit/DisplayName.js

```diff
@@ -14,7 +14,6 @@ export const DisplayName = (props, context) => {
   const InputComponent = FUNDAMENTAL_DATA_TYPES[port.type || 'any'];
 
   const hasInput = !isOutput
-    && port.connected_to === NULL_REF
```

Comment:
you should make sure it isn't connected to anything.

Reply:
this is necessary to prevent the input fields for hardcoding ports from appearing when there is already a connection